site/docs/v1/tech/apis/identity-providers/links.adoc

```diff
@@ -66,11 +68,51 @@ include::docs/v1/tech/apis/_standard-post-response-codes.adoc[]
 
 include::docs/v1/tech/apis/identity-providers/_links-post-response-body.adoc[]
 
+=== Request
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Link a User
+[.endpoint]
+.URI
+--
+[method]#POST# [uri]#/api/identity-provider/link#
+--
+[WARNING.deprecated]
+====
+While this endpoint is still functional, its usage is discouraged. Please use the link:/docs/v1/tech/apis/identity-providers/links#link-a-user[Link a User endpoint] which expects an `identityProviderLink` object if your version of FusionAuth is 1.43.0 or above.
+====
+
+==== Request Body
+
+[.api]
+[field]#displayName# [type]#[String]# [optional]#Optional# [since]#Available since 1.28.1#::
+A human-readable name for this link when initially created. This value should be used to make it easier to identify the user this link represents in the remote identity provider.
++
+After the link is created, and a user successfully logs in for the first time with the IdP, this value will typically be set to an email or username.
+
+[field]#identityProviderId# [type]#[UUID]# [required]#Required#::
+The Id of the identity provider. This identity provider must exist.
+
+[field]#identityProviderUserId# [type]#[String]# [required]#Required#::
+The Id for the User that is provided by the identity provider. This is the value that will allow FusionAuth to link this user on future logins. It is expected to be immutable.
+
+[field]#userId# [type]#[UUID]# [required]#Required#::
+The FusionAuth Id of the User that is being linked to the identity provider.
+
```

Comment:
Should these deprecated fields just be left in the original API doc and marked as deprecated instead of making a separate request?